packages/slice-machine/pages/onboarding.tsx

```diff
@@ -83,32 +102,39 @@ const WelcomeSlide = ({ onClick }: { onClick: () => void }) => {
     </>
   );
 };
-const BuildSlicesSlide = () => (
+const BuildSlicesSlide: React.FC<{ version: string; framework: Frameworks }> = (
+  props
+) => (
   <>
     <Image sx={imageSx} src="/horizontal_split.svg" />
     <Header>Build Slices</Header>
     <SubHeader>The building blocks used to create your website</SubHeader>
-    <Video publicId="SMONBOARDING/BUILD_SLICE" />
+    <Video {...props} publicId={VIDEO_ONBOARDING_BUILD_A_SLICE} />
   </>
 );
 
-const CreatePageTypesSlide = () => (
+const CreatePageTypesSlide: React.FC<{
+  version: string;
+  framework: Frameworks;
+}> = (props) => (
   <>
     <Image sx={imageSx} src="/insert_page_break.svg" />
     <Header>Create Page Types</Header>
     <SubHeader>Group your Slices as page builders</SubHeader>
-    <Video publicId="SMONBOARDING/ADD_TO_PAGE" />
+    <Video {...props} publicId={VIDEO_ONBOARDING_ADD_TO_PAGE} />
   </>
 );
 
-const PushPagesSlide = () => (
+const PushPagesSlide: React.FC<{ version: string; framework: Frameworks }> = (
+  props
+) => (
   <>
     <Image sx={imageSx} src="/send.svg" />
     <Header>Push your pages to Prismic</Header>
     <SubHeader>
       Give your content writers the freedom to build whatever they need
     </SubHeader>
-    <Video publicId="SMONBOARDING/PUSH_TO_PRISMIC" />
+    <Video {...props} publicId={VIDEO_ONBOARDING_PUSH_TO_PRISMIC} />
```

Comment:
why do you need to provide **version** and **framework** to the cloudinary component?